### PR TITLE
Add circleci configuration

### LINKED ISSUE #76: Setup CircleCI
Add CircleCI

### PR DESCRIPTION
**Pull request by jondequinor**
Fixes #76

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+  python_build:
+    parameters:
+      python_version:
+        type: string
+        default: ""
+    steps:
+      - checkout
+      - run: sudo chown -R circleci:circleci /usr/local/bin
+      - run: sudo chown -R circleci:circleci /usr/local/lib/python<< parameters.python_version >>/site-packages
+      - run: cat requirements/* > requirements.lock
```
Comment (markusdregi) [inline .circleci/config.yml]: Would not `cat requirements/circleci-* > requirements.lock` be more accurate?

**Comment by andreabrambilla** [comment]
Not strictly relates to this pr, as I see the same thing is done for Travis.
Why do we need sets of separate requirements for Travis and circleci? 
Can't we rely on what's defined in setup.py?

**Comment by andreabrambilla** [comment]
Nevermind, I've just seen issue #82

File: .circleci/config.yml (hunk 1 above)
Comment (jondequinor) [reply .circleci/config.yml]: Addressed in a302e9f.

**Pull request merged**